proposed/http-middleware/middleware-meta.md

```diff
@@ -222,8 +222,8 @@ and a delegate and must return a response. The middleware may:
 - Evolve the request before passing it to the delegate to execute the next
   available middleware.
 - Evolve the response received from the delegate before returning it.
-- Create and return a response without passing it to the delegate, thereby
-  preventing any further middleware from processing.
+- Create and return a response without further process delegation, thereby
```

Comment:
Maybe I read that wrong, but I believe this means `a/the process (noun)` will not be delegated. But what `process`? The `request process`?